CHANGELOG.md

```diff
@@ -7,8 +7,9 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 - Ability to add tooltips to `Slider` and `RangeSlider`, which can be visible always or on hover. Tooltips also take a position argument. [#564](https://github.com/plotly/dash-core-components/pull/564)
 
 ### Fixed
-- Fixed `min_date_allowed` and `max_date_allowed` bug in `DatePickerRange` [#551]https://github.com/plotly/dash-core-components/issues/551)
+- Fixed `min_date_allowed` and `max_date_allowed` bug in `DatePickerRange` [#551](https://github.com/plotly/dash-core-components/issues/551)
```

Comment:
hah cc @wbrgss I thought you already fixed this, bad rebase in #564?

Reply:
Yeah, I thought I fixed it, must be.